lib/redcord/attribute.rb

```diff
@@ -79,5 +94,26 @@ def should_range_index?(type)
     end
   end
 
+  module InstanceMethods
+    extend T::Sig
+
+    sig { returns(T.nilable(String)) }
+    def hash_tag
+      attr = self.class.class_variable_get(:@@shard_by_attribute)
+
+      return nil if attr.nil?
+
+      # A blank hash tag would cause MOVED error in cluster mode
+      tag = send(attr)
+      default_tag = '__redcord_hash_tag_null__'
+
+      if tag == default_tag
+        raise "#{attr}=#{default_tag} conflict with default hash_tag value"
```

Comment:
this would turn out to a RuntimeError. I think this is fine because Redcord specific error is only useful when people want to catch them programmatically